servers/src/main/java/tachyon/master/file/FileSystemMasterServiceHandler.java

```diff
@@ -98,9 +99,9 @@ public String getUfsAddress() {
   }
 
   @Override
-  public long createFile(String path, long blockSizeBytes, boolean recursive)
+  public long createFile(String path, long blockSizeBytes, boolean recursive, long ttl)
```

Comment:
Could you leave a TODO saying that there should be an options object used here? Thanks!